### PR TITLE
Download bpmn files as images

### PR DESCRIPTION
**Pull request by MartijnBogaert**
Corresponding PRs:
- bpmn-service: https://github.com/lblod/bpmn-service/pull/4
- frontend-openproceshuis: https://github.com/lblod/frontend-openproceshuis/pull/29

BPMN files can now be served in different formats. The download URL is always the same (`files/:id/download`), but the returned file's format differs based on the acccept header's mime type. BPMN files in their original form don't need conversion and are simply served by the file service, while image based types are served by the BPMN service.

### CODE REVIEW

File: config/dispatcher/dispatcher.ex [hunk 1]
```diff
@@ -29,7 +31,13 @@ defmodule Dispatcher do
...
   ###############################################################
 
-  get "/files/:id/download", %{ layer: :api_services } do
+  get "/files/:id/download", %{ accept: %{ bpmn_converted: true }, layer: :api_services } do
+    IO.inspect( conn, label: "BPMN converted" )
+    Proxy.forward conn, [], "http://bpmn/" <> id <> "/download"
+  end
+
+  get "/files/:id/download", %{ accept: %{ bpmn_original: true }, layer: :api_services } do
+    IO.inspect( conn, label: "BPMN original" )
```
Comment (nbittich) [inline config/dispatcher/dispatcher.ex:40]: is it still needed?

File: config/dispatcher/dispatcher.ex [hunk 2]
```diff
@@ -29,7 +31,13 @@ defmodule Dispatcher do
   # files
   ###############################################################
 
-  get "/files/:id/download", %{ layer: :api_services } do
+  get "/files/:id/download", %{ accept: %{ bpmn_converted: true }, layer: :api_services } do
+    IO.inspect( conn, label: "BPMN converted" )
```
Comment (nbittich) [inline config/dispatcher/dispatcher.ex:35]: is it still needed?

**Review by nbittich — COMMENTED** [review]
looks ok except my small comments, label was missing tho :P

**Review by MartijnBogaert — COMMENTED** [review]
> looks ok except my small comments, label was missing tho :P 

thanks for noticing, should be resolved now

**Review by nbittich — APPROVED** [review]

**Pull request merged**